src/shared/vscode/commands2.ts

```diff
@@ -218,8 +231,7 @@ interface Builder {
 }
 
 interface Deferred<T extends Callback, U extends any[]> {
-    readonly id: string
-    readonly name?: string
+    readonly info: CommandInfo<T>
     readonly factory: CommandFactory<T, U>
```

Comment:
idk quite what the purpose of `Deferred` is, except to associate a factory with a command?